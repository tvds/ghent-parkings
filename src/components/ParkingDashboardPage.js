import React from 'react';
import { connect } from 'react-redux';
import ParkingList from './ParkingList';
import Map from "./Map";

export class ParkingDashboardPage extends React.Component {
  render() {
    return (
      <div>
        <div className='dashboard-left'>
          <ParkingList parkings={this.props.parkings} />
        </div>
        <div className='dashboard-right'>
          <Map
            center={{ lat: 51.052757, lng: 3.718902 }}
            zoom={13}
            places={this.props.parkings}
          />
        </div>
      </div>
    );
  };
}

const mapStateToProps = (state) => {
  return {
    parkings: state.parkings
  };
};

export default connect(mapStateToProps)(ParkingDashboardPage);
