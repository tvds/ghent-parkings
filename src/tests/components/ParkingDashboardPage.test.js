import React from 'react';
import { shallow } from 'enzyme';
import ParkingDashboardPage from '../../components/ParkingDashboardPage';

test('should render ParkingDashboardPage correctly', () => {
  const wrapper = shallow(<ParkingDashboardPage />);
  expect(wrapper).toMatchSnapshot();
});
