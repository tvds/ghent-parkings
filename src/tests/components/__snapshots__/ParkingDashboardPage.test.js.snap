// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ParkingDashboardPage correctly 1`] = `<[object Object] />`;
